testpilot-metrics.js

```diff
@@ -24,15 +24,15 @@
  */
 function Metrics({id, version, uid, tid = null, type = 'webextension', debug = false}) {
   if (!id) {
-    throw new SyntaxError(`'id' property is required.`);
+    throw new SyntaxError("'id' property is required.");
```

Comment:
I dunno, I kinda wanted to just use template strings all the time. Is there an eslint setting for that?